drivers/ccpp/catchem_dust_wrapper.F90

```diff
@@ -189,18 +190,22 @@ subroutine catchem_dust_wrapper_run(im, kte, kme, ktau, dt, garea, land,   &
         do j=jts,jte
           do i=its,ite
             if(xland(i,j).lt.1.5)then ! JianHe: why not also > 0.5?
-              delp = p8w(i,kts,j)-p8w(i,kts+1,j)            
-              ! -- GOCART afwa dust scheme
+              delp = p8w(i,kts,j)-p8w(i,kts+1,j)    
+              airmas = dxy(i,j) * delp / g        
+              ! -- afwa dust scheme
               call gocart_dust_afwa_driver(ktau,dt,u_phy(i,kts,j), &
-                v_phy(i,kts,j),chem(i,kts,j,:),rho_phy(i,kts,j), &
+                v_phy(i,kts,j),rho_phy(i,kts,j), &
                 dz8w(i,kts,j),smois(i,:,j),u10(i,j), &
                 v10(i,j),delp,erod(i,j,:),isltyp(i,j),dxy(i,j),    &
                 emis_dust(i,1,j,:),&
                 ust(i,j),znt(i,j),clayf(i,j),sandf(i,j),nsoil)
               !store_arrays = .true.
-            endif
-          enddo
-        enddo
+              do n=0,ndust-1
```

Comment:
duplicated for each opt, maybe collapse to one at end of case select?

Reply:
The other two don't have the RHS `dxy`, I notice.

Reply:
There is a slight difference.  I suggest we move forward to get this in and have a subsequent update this going forward.